### PR TITLE
Fix statusLabel error when launching OCR

### PR DESCRIPTION
**Pull request by christian-oreilly**
Fix the following error message that was happening when launching OCR:

Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/home/oreilly/.local/lib/python3.6/site-packages/neurocurator/mainWin.py", line 1084, in waitForOCR
    self.window.statusLabel.setText("Performing OCR...")
AttributeError: 'builtin_function_or_method' object has no attribute 'statusLabel'

### CODE REVIEW

**Comment by pafonta** [comment]
@christian-oreilly Could you provide a reference ID to reproduce the issue and then to verify if it has been fixed? I don't find one which need OCR after having looked for several references.

**Comment by christian-oreilly** [comment]
I already did that these check @pafonta, else I would not have submitted a PR. But anyway, if you want to double check, any ID will do fine (it does not depend on a particular ID). What you need is a PDF that has no text so that adding it will trigger the OCR process. That, I can provide some copy offline (for copyright reasons). If you want to have it documented publicly, then a solution is to take a copy-left paper (e.g., a Frontiers one), turn it into an image pdf, and then submit. Let me know how you prefer to proceed.

File: neurocurator/mainWin.py [hunk 1]
```diff
@@ -1081,10 +1081,10 @@ def clearPaper(self):
     def waitForOCR(self, paperId, notify):
         
         while(not self.restClient.checkOCRFinished(paperId, self.dbPath)):
-            self.window.statusLabel.setText("Performing OCR...")
+            self.statusBar().showMessage("Performing OCR...")
             time.sleep(5)          
         
-        self.window.statusLabel.setText("OCR finished.")
+        self.statusBar().showMessage("OCR finished.")
```
Comment (pafonta) [inline neurocurator/mainWin.py]: Should use argument `timeout` [doc](https://doc.qt.io/qt-5/qstatusbar.html#showMessage). I used 10 * 1000 elsewhere (Zotero database refresh).

Comment (christian-oreilly) [reply neurocurator/mainWin.py]: Well, if you want to refactor these things, sure. This PR was just about fixing the error with the use of the statusLabel widget. The rest is currently unchanged.

**Comment by pafonta** [comment]
> Let me know how you prefer to proceed.
Just having the NeuroCurator type ID of the reference so that I can test this issue and others that implies PDFs needing OCR. I am thinking about the one about the not created PCR file or testing OCRmyPDF. I don't need the PDF nor that it could be public, just the ID because for now I have just don't have a reference for which we now the PDF need OCR.

**Pull request merged**